plotly/tests/test_optional/test_utils/test_utils.py

```diff
@@ -96,20 +96,23 @@ def test_encode_as_numpy(self):
         res = utils.PlotlyJSONEncoder.encode_as_numpy(np.ma.core.masked)
         self.assertTrue(math.isnan(res))
 
-    def test_encode_as_datetime(self):
+    def test_encode_valid_datetime(self):
 
         # should *fail* without 'utcoffset' and 'isoformat' and '__sub__' attrs
-        non_datetimes = [datetime.date(2013, 10, 1), 'noon', 56, '00:00:00']
+        #non_datetimes = [datetime.date(2013, 10, 1), 'noon', 56, '00:00:00']
+        non_datetimes = [datetime.date(2013, 10, 1)]
```

Comment:
So Python3.6 handles datetimes differently than the other Python versions - and since we were not testing Py36 before it's resulting in a failing test now